features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1749,6 +1757,18 @@
                        orderId="{{event.default_listener.client_certificate_authentication_handler.priority}}"
                        enable="{{event.default_listener.client_certificate_authentication_handler.enable}}"/>
 
+        <!-- B2B organization management Event Listeners -->
+        <EventListener id="fragment_app_mgt_listener"
+                               type="org.wso2.carbon.identity.application.mgt.listener.ApplicationMgtListener"
+                               name="org.wso2.carbon.identity.organization.management.application.listener.FragmentApplicationMgtListener"
+                               orderId="{{event.default_listener.fragment_app_mgt_listener.priority}}"
+                               enable="{{event.default_listener.fragment_app_mgt_listener.enable}}"/>
+
+        <EventListener id="organization_user_operation_listener"
+                                       type="org.wso2.carbon.user.core.listener.UserOperationEventListener"
+                                       name="org.wso2.carbon.identity.organization.management.role.management.service.listener.OrganizationUserOperationEventListener"
+                                       orderId="{{event.default_listener.organization_user_operation_listener.priority}}"
+                                       enable="{{event.default_listener.organization_user_operation_listener.enable}}"/>
```

Comment:
Shall we align all the params against the `id` parameter? (Fix the indentation)

Reply:
Fixed by - https://github.com/wso2/carbon-identity-framework/pull/4825/commits/8433f50d175e8247aa74cb121e4bf2dc06f7e4d8